watchtower/wtclient/backup_task.go

```diff
@@ -0,0 +1,344 @@
+package wtclient
+
+import (
+	"github.com/btcsuite/btcd/blockchain"
+	"github.com/btcsuite/btcd/btcec"
+	"github.com/btcsuite/btcd/txscript"
+	"github.com/btcsuite/btcd/wire"
+	"github.com/btcsuite/btcutil"
+	"github.com/btcsuite/btcutil/txsort"
+	"github.com/lightningnetwork/lnd/input"
+	"github.com/lightningnetwork/lnd/lnwallet"
+	"github.com/lightningnetwork/lnd/lnwire"
+	"github.com/lightningnetwork/lnd/watchtower/blob"
+	"github.com/lightningnetwork/lnd/watchtower/wtdb"
+)
+
+// backupTask is an internal struct for computing the justice transaction for a
+// particular revoked state. A backupTask functions as a scratch pad for storing
+// computing values of the transaction itself, such as the final split in
+// balance if the justice transaction will give a reward to the tower. The
+// backup task has three primary phases:
+//  1. Init: Determines which inputs from the breach transaction will be spent,
+//     and the total amount contained in the inputs.
+//  2. Bind: Asserts that the revoked state is eligible under a given session's
+//     parameters. Certain states may be ineligible due to fee rates, too little
+//     input amount, etc.. Backup of these states can be deferred to a later
```

Comment:
Extra period here after "etc"?

Reply:
heh, i've always thought it was correct to add another period at the end of a sentence, apparently not so. it is okay to add a comma after tho. english yo 🤷‍♂️